README
NAME
    Dist::Zilla::PluginBundle::DROLSKY - DROLSKY's plugin bundle

VERSION
    version 0.05

AUTHOR
    Dave Rolsky <autarch@urth.org>

COPYRIGHT AND LICENSE
    This software is Copyright (c) 2013 by Dave Rolsky.

    This is free software, licensed under:

      The Artistic License 2.0 (GPL Compatible)



Changes
0.27     2014-12-20

- Replace GatherDir and PruneCruft with Git::GatherDir.


0.26     2014-11-15

- The generated Makefile.PL or Build.PL file is now copied into the repo and
  checked in for each release.


0.25     2014-11-09

- This bundle now consumes the Dist::Zilla::Role::PluginBundle::Config::Slicer
  role to allow for arbitrary plugin configuration.


0.24     2014-11-04

- Make sure that a copyright_holder in dist.ini overrides the default of
  "David Rolsky".


0.23     2014-11-02

- Oops, accidentally removed the CopyFilesFromBuild plugin from the prereq
  list in the last release.


0.22     2014-11-02

- The DROLSKY::Contributors plugin now leaves existing .mailmap and weaver.ini
  files alone.

- Exclude plugins shipped with this bundle when checking for stale modules.

- Generate a cpanfile and copy it into the repo after a build.

- Copy the LICENSE file into the repo after abuild.


0.21     2014-11-01

- Added custom plugins to setup the weaver.ini file for Contributors and to
  automatically provide default license info.


0.20     2014-11-01

- Explicitly depend on Dist::Zilla::Plugin::PodWeaver because
  SurgicalPodWeaver doesn't - works around RT #87624.


0.19     2014-10-23

- Switched to Test::EOL (0.14)

- Make PkgVersion use a bit smarter (no line insertion and die on existing
  $VERSION in module)

- Update next release version to indicate if a release is a trial


0.18     2014-09-03

- Switched from ContributorsFromGit to Git::Contributors.


0.17     2014-08-15

- Renamed the attributes but didn't fix the method calls in the last version.


0.16     2014-08-15

- Renamed some options - coverage_* => pod_coverage_*


0.15     2014-08-12

- Ordering fix take 2.


0.14     2014-08-12

- Hopefully fix an ordering issue with the Git plugins.


0.13     2014-08-12

- Use GatherDir to exclude files rather than PruneFiles.


0.12     2014-08-03

- Added Test::Pod::Coverage::Configurable plugin.

- Removed the remove option in favor of -remove.


0.11     2014-08-02

- Added GitHub::Update plugin.


0.10     2014-08-02

- Added a number of additional plugins to help me make better dists.

- Moved this distro to GitHub.


0.09     2014-05-04

- Use Test::NoTabs plugin instead of NoTabsTests.


0.08     2014-04-05

- Internals fixes to work with Config::MVP that is stricter about handling
  plugin options passed by this bundle. Reported by Karen Etheridge. RT
  #94466.

- Make sure all the plugins this bundle uses end up declared as
  prereqs. Reported by Dagfinn Ilmari Mannsåker. RT #94363.


0.07     2014-01-25

- Added the PruneFiles plugin to prune the README file that gets copied from
  the build.

- Allow the README to be dirty for Git::Commit.


0.06     2014-01-25

- Added "do_munging => 0" to the options for the Authority plugin so it
  doesn't add a useless $AUTHORITY var to packages.


0.05     2013-12-13

- Added ContributorsFromGit and Meta::Contributors plugins.


0.04     2013-12-13

- Added more plugins and support for removing plugins.


0.03     2013-06-07

- Allow multiple prereqs_skip and stopwords properties


0.02     2013-05-13

- Allow prereqs_skip option to be passed to AutoPrereqs


0.01     2013-04-21

- First release upon an unsuspecting world.


Makefile.PL

# This file was automatically generated by Dist::Zilla::Plugin::MakeMaker v5.024.
use strict;
use warnings;

use 5.010000;

use ExtUtils::MakeMaker;



my %WriteMakefileArgs = (
  "ABSTRACT" => "DROLSKY's plugin bundle",
  "AUTHOR" => "Dave Rolsky <autarch\@urth.org>",
  "CONFIGURE_REQUIRES" => {
    "ExtUtils::MakeMaker" => 0
  },
  "DISTNAME" => "Dist-Zilla-PluginBundle-DROLSKY",
  "EXE_FILES" => [],
  "LICENSE" => "artistic_2",
  "MIN_PERL_VERSION" => "5.010000",
  "NAME" => "Dist::Zilla::PluginBundle::DROLSKY",
  "PREREQ_PM" => {
    "Dist::Zilla" => 0,
    "Dist::Zilla::File::InMemory" => 0,
    "Dist::Zilla::Plugin::Authority" => 0,
    "Dist::Zilla::Plugin::AutoPrereqs" => 0,
    "Dist::Zilla::Plugin::CPANFile" => 0,
    "Dist::Zilla::Plugin::CheckPrereqsIndexed" => 0,
    "Dist::Zilla::Plugin::CopyFilesFromBuild" => 0,
    "Dist::Zilla::Plugin::Git::Check" => 0,
    "Dist::Zilla::Plugin::Git::CheckFor::MergeConflicts" => 0,
    "Dist::Zilla::Plugin::Git::Commit" => 0,
    "Dist::Zilla::Plugin::Git::Contributors" => 0,
    "Dist::Zilla::Plugin::Git::Describe" => 0,
    "Dist::Zilla::Plugin::Git::Push" => 0,
    "Dist::Zilla::Plugin::Git::Tag" => 0,
    "Dist::Zilla::Plugin::GitHub::Meta" => 0,
    "Dist::Zilla::Plugin::GitHub::Update" => 0,
    "Dist::Zilla::Plugin::InstallGuide" => 0,
    "Dist::Zilla::Plugin::Meta::Contributors" => 0,
    "Dist::Zilla::Plugin::MetaConfig" => 0,
    "Dist::Zilla::Plugin::MetaJSON" => 0,
    "Dist::Zilla::Plugin::MetaProvides::Package" => 0,
    "Dist::Zilla::Plugin::MetaResources" => 0,
    "Dist::Zilla::Plugin::MojibakeTests" => 0,
    "Dist::Zilla::Plugin::NextRelease" => 0,
    "Dist::Zilla::Plugin::PkgVersion" => 0,
    "Dist::Zilla::Plugin::PodSyntaxTests" => 0,
    "Dist::Zilla::Plugin::PodWeaver" => 0,
    "Dist::Zilla::Plugin::PromptIfStale" => 0,
    "Dist::Zilla::Plugin::ReadmeAnyFromPod" => 0,
    "Dist::Zilla::Plugin::SurgicalPodWeaver" => 0,
    "Dist::Zilla::Plugin::Test::CPAN::Changes" => 0,
    "Dist::Zilla::Plugin::Test::Compile" => 0,
    "Dist::Zilla::Plugin::Test::EOL" => "0.14",
    "Dist::Zilla::Plugin::Test::NoTabs" => 0,
    "Dist::Zilla::Plugin::Test::Pod::Coverage::Configurable" => 0,
    "Dist::Zilla::Plugin::Test::Pod::LinkCheck" => 0,
    "Dist::Zilla::Plugin::Test::Pod::No404s" => 0,
    "Dist::Zilla::Plugin::Test::PodSpelling" => 0,
    "Dist::Zilla::Plugin::Test::Portability" => 0,
    "Dist::Zilla::Plugin::Test::ReportPrereqs" => 0,
    "Dist::Zilla::Plugin::Test::Synopsis" => 0,
    "Dist::Zilla::Role::AfterBuild" => 0,
    "Dist::Zilla::Role::BeforeBuild" => 0,
    "Dist::Zilla::Role::LicenseProvider" => 0,
    "Dist::Zilla::Role::PluginBundle::Config::Slicer" => 0,
    "Dist::Zilla::Role::PluginBundle::Easy" => 0,
    "Dist::Zilla::Role::PluginBundle::PluginRemover" => 0,
    "Module::Runtime" => 0,
    "Moose" => 0,
    "Pod::Weaver::Section::Contributors" => 0,
    "String::RewritePrefix" => 0,
    "autodie" => 0,
    "strict" => 0,
    "warnings" => 0
  },
  "TEST_REQUIRES" => {
    "ExtUtils::MakeMaker" => 0,
    "File::Spec" => 0,
    "IO::Handle" => 0,
    "IPC::Open3" => 0,
    "Test::More" => "0.88"
  },
  "VERSION" => "0.25",
  "test" => {
    "TESTS" => "t/*.t"
  }
);


my %FallbackPrereqs = (
  "Dist::Zilla" => 0,
  "Dist::Zilla::File::InMemory" => 0,
  "Dist::Zilla::Plugin::Authority" => 0,
  "Dist::Zilla::Plugin::AutoPrereqs" => 0,
  "Dist::Zilla::Plugin::CPANFile" => 0,
  "Dist::Zilla::Plugin::CheckPrereqsIndexed" => 0,
  "Dist::Zilla::Plugin::CopyFilesFromBuild" => 0,
  "Dist::Zilla::Plugin::Git::Check" => 0,
  "Dist::Zilla::Plugin::Git::CheckFor::MergeConflicts" => 0,
  "Dist::Zilla::Plugin::Git::Commit" => 0,
  "Dist::Zilla::Plugin::Git::Contributors" => 0,
  "Dist::Zilla::Plugin::Git::Describe" => 0,
  "Dist::Zilla::Plugin::Git::Push" => 0,
  "Dist::Zilla::Plugin::Git::Tag" => 0,
  "Dist::Zilla::Plugin::GitHub::Meta" => 0,
  "Dist::Zilla::Plugin::GitHub::Update" => 0,
  "Dist::Zilla::Plugin::InstallGuide" => 0,
  "Dist::Zilla::Plugin::Meta::Contributors" => 0,
  "Dist::Zilla::Plugin::MetaConfig" => 0,
  "Dist::Zilla::Plugin::MetaJSON" => 0,
  "Dist::Zilla::Plugin::MetaProvides::Package" => 0,
  "Dist::Zilla::Plugin::MetaResources" => 0,
  "Dist::Zilla::Plugin::MojibakeTests" => 0,
  "Dist::Zilla::Plugin::NextRelease" => 0,
  "Dist::Zilla::Plugin::PkgVersion" => 0,
  "Dist::Zilla::Plugin::PodSyntaxTests" => 0,
  "Dist::Zilla::Plugin::PodWeaver" => 0,
  "Dist::Zilla::Plugin::PromptIfStale" => 0,
  "Dist::Zilla::Plugin::ReadmeAnyFromPod" => 0,
  "Dist::Zilla::Plugin::SurgicalPodWeaver" => 0,
  "Dist::Zilla::Plugin::Test::CPAN::Changes" => 0,
  "Dist::Zilla::Plugin::Test::Compile" => 0,
  "Dist::Zilla::Plugin::Test::EOL" => "0.14",
  "Dist::Zilla::Plugin::Test::NoTabs" => 0,
  "Dist::Zilla::Plugin::Test::Pod::Coverage::Configurable" => 0,
  "Dist::Zilla::Plugin::Test::Pod::LinkCheck" => 0,
  "Dist::Zilla::Plugin::Test::Pod::No404s" => 0,
  "Dist::Zilla::Plugin::Test::PodSpelling" => 0,
  "Dist::Zilla::Plugin::Test::Portability" => 0,
  "Dist::Zilla::Plugin::Test::ReportPrereqs" => 0,
  "Dist::Zilla::Plugin::Test::Synopsis" => 0,
  "Dist::Zilla::Role::AfterBuild" => 0,
  "Dist::Zilla::Role::BeforeBuild" => 0,
  "Dist::Zilla::Role::LicenseProvider" => 0,
  "Dist::Zilla::Role::PluginBundle::Config::Slicer" => 0,
  "Dist::Zilla::Role::PluginBundle::Easy" => 0,
  "Dist::Zilla::Role::PluginBundle::PluginRemover" => 0,
  "ExtUtils::MakeMaker" => 0,
  "File::Spec" => 0,
  "IO::Handle" => 0,
  "IPC::Open3" => 0,
  "Module::Runtime" => 0,
  "Moose" => 0,
  "Pod::Weaver::Section::Contributors" => 0,
  "String::RewritePrefix" => 0,
  "Test::More" => "0.88",
  "autodie" => 0,
  "strict" => 0,
  "warnings" => 0
);


unless ( eval { ExtUtils::MakeMaker->VERSION(6.63_03) } ) {
  delete $WriteMakefileArgs{TEST_REQUIRES};
  delete $WriteMakefileArgs{BUILD_REQUIRES};
  $WriteMakefileArgs{PREREQ_PM} = \%FallbackPrereqs;
}

delete $WriteMakefileArgs{CONFIGURE_REQUIRES}
  unless eval { ExtUtils::MakeMaker->VERSION(6.52) };

WriteMakefile(%WriteMakefileArgs);



